src/demo-app/a11y/toolbar/toolbar-a11y.html

```diff
@@ -0,0 +1,28 @@
+<div class="demo-toolbar">
+  <section>
+    <h2>Display</h2>
+    <md-toolbar>My App</md-toolbar>
+  </section>
+
+  <section>
+    <h2>Multiple Lines Toolbar</h2>
+    <md-toolbar>Line 1</md-toolbar>
+    <md-toolbar>Line 2</md-toolbar>
+  </section>
+
+  <section>
+    <h2>Toolbar with Icons</h2>
+    <md-toolbar>
+      <span>My App</span>
+      <span class="example-spacer"></span>
+      <md-icon class="example-icon">favorite</md-icon>
```

Comment:
Icons actually are `aria-hidden` by default. They should only be made `aria-hidden="false"` if they're communicating some information. See the accessibility section of [the icon docs](https://material.angular.io/components/icon/overview). 

In this case, having an icon sitting by itself in the toolbar isn't very realistic to a real app; it would either be an indicator for something or an action.